modules/accredited_representative_portal/app/controllers/accredited_representative_portal/v0/power_of_attorney_requests_controller.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module AccreditedRepresentativePortal
+  module V0
+    class PowerOfAttorneyRequestsController < ApplicationController
+      # TODO: address authorization
+
+      def show
+        poa_request_details_id = params[:id]
+        poa_request_details_service = PoaRequestDetailsService.new(poa_request_details_id)
+        render json: poa_request_details_service.call
+      end
```

Comment:
What do you think about starting with everything inlined? The reason I suggest this is because while I do think a services layer is the right place for business logic, I have these 2 points:
- We could prefer a different convention for organizing that layer, especially something like: `context/action` as opposed to `context_action` like we have here
- When a controller is genuinely absent of any nontrivial business logic, I prefer not to introduce an empty intermediate layer to just have less code

Reply:
sounds great! I love YAGNI. Will change 👍